### PR TITLE
Adding language Arabic

### PR DESCRIPTION
**Pull request by Hamad3bdulla**
Adding a new language, Arabic

The translation has been added and the previous errors have been corrected. 
I hope it now works successfully. 
Thank you

### CODE REVIEW

File: src/lib/i18n/locales/ar-BH/translation.json [hunk 1]
```diff
@@ -0,0 +1,443 @@
+{
```
Comment (tjbck) [inline src/lib/i18n/locales/ar-BH/translation.json:1]: translation.json should look something like this: https://github.com/open-webui/open-webui/blob/main/src/lib/i18n/locales/bg-BG/translation.json

Comment (bo3bdo) [reply src/lib/i18n/locales/ar-BH/translation.json:1]: hahaha oh sorry my mistake now I understand

Comment (bo3bdo) [reply src/lib/i18n/locales/ar-BH/translation.json:1]: fix Thank you for your help @tjbck

**Comment by tjbck** [comment]
LGTM, Thanks!

**Comment by bo3bdo** [comment]
Thanks Timothy

**Pull request merged**